M2/Macaulay2/packages/SimplicialComplexes/Documentation.m2

```diff
@@ -3035,7 +3035,7 @@ doc ///
 	    prune homology Γ
 	    prune homology(Γ, QQ)
 	    prune homology(Γ, ZZ/2)
-	    assert(prune homology(Γ, ZZ/2) == gradedModule((ZZ/2)^2[-1] ++ (ZZ/2)^1[-2]))
+	    assert(prune homology(Γ, ZZ/2) == (complex (ZZ/2)^2)[-1] ++ (complex (ZZ/2)^1)[-2])
```

Comment:
If the plan is to entirely remove `gradedModule`, and change `M[n]` to return a `Complex`, this would just become `(ZZ/2)^2[-1] ++ (ZZ/2)^1[-2]`, correct?